src/Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG PYTHON_VERSION=3.8
+FROM python:${PYTHON_VERSION} as builder
+
+WORKDIR /app
+
+COPY . .
+
+FROM python:${PYTHON_VERSION}-slim
+
+WORKDIR /app
+
+COPY --from=builder /app .
+
+RUN pip install --upgrade pip && \
+    pip install -r requirements.txt
+
+RUN python manage.py migrate
+
+EXPOSE 8080
+
+CMD [ "python", "manage.py", "runserver", "0.0.0.0:8080" ]
```

Comment:
![image](https://github.com/user-attachments/assets/ef39d629-27ae-4c08-b3e7-cb7b5cbaa988)
Add Eof(End-of-file), need to add blank line at the end